app/src/androidTest/java/org/mozilla/focus/autobot/SessionRobot.kt

```diff
@@ -81,16 +82,43 @@ class SessionRobot : MenuRobot() {
 
         runWithIdleRes(screenshotIdlingResource) {
             AndroidTestUtils.toastContainsText(activity, R.string.screenshot_saved)
+            checkNoScreenshotOnBoarding()
         }
     }
 
+    fun checkNoScreenshotOnBoarding() {
+        onView(Matchers.allOf(withId(R.id.my_shot_img_hand))).check(doesNotExist())
+        onView(Matchers.allOf(withId(R.id.my_shot_category_learn_more))).check(doesNotExist())
+    }
+
+    fun firstTimeclickCaptureScreen(activity: MainActivity) {
+        // TODO find a way to remove the activity reference
+        screenshotIdlingResource = ScreenshotIdlingResource(activity)
+
+        // Click screen capture button
+        onView(Matchers.allOf(withId(R.id.btn_capture), isDisplayed())).perform(click())
+        checkScreenshotOnBoarding()
```

Comment:
why need a sperate method?

Reply:
this method is used as public and private so need separate one